Formula/jolie.rb

```diff
@@ -0,0 +1,21 @@
+class Jolie < Formula
+  desc "The Jolie Language Interpreter"
+  homepage "https://www.jolie-lang.org/"
+  url "https://github.com/jolie/jolie/releases/download/v1.9.0/jolie-1.9.0.jar"
+  sha256 "1510ed7f114909eb79670462571ab0734a5b01e57d26da6fd1cf9ef6c67eff6e"
+
+  depends_on "openjdk"
+
+  def install
+    system "java", "-jar", "jolie-#{version}.jar", "-jh", libexec, "-jl", bin
```

Comment:
something like here might be better: https://github.com/Homebrew/homebrew-core/blob/master/Formula/jruby.rb

using a wrapper around the jar file that sets `JOLIE_HOME` and `JAVA_HOME`

Reply:
Thanks for the pointer :)
If I understood it correctly, the wrapper would only work for the builder but not for the commands, which in any case would require the $JOLIE_HOME env variable set.
In the installer, I am avoiding this by replacing the JOLIE_HOME with the path to libexec, but if there is a guideline followed in brew I'd gladly follow it.
What is the preferred direction? Setting an env variable? If yes, do you have a pointer for that?

Reply:
The preferred way is to use a wrapper script that sets the environment variables. Ideally we don't edit any files from the developers.

Reply:
Replace env. replacement with wrappers